azure-toolkit-libs/pom.xml

```diff
@@ -54,62 +54,67 @@
-        <guava.version>30.1-jre</guava.version>
-        <httpclient.version>4.5.9</httpclient.version>
-        <jackson.version>2.12.1</jackson.version>
+        <error.prone.core.version>2.7.1</error.prone.core.version>
+        <gson.version>2.8.7</gson.version>
+        <guava.version>30.1.1-jre</guava.version>
+        <httpclient.version>4.5.13</httpclient.version>
+        <jackson.version>2.12.3</jackson.version>
+        <jacoco.version>0.8.7</jacoco.version>
         <jacoco.version>0.8.5</jacoco.version>
-        <jansi.version>1.17.1</jansi.version>
-        <jetbrains.annotations.version>19.0.0</jetbrains.annotations.version>
+        <jansi.version>2.3.2</jansi.version>
+        <jetbrains.annotations.version>21.0.1</jetbrains.annotations.version>
         <json.schema.validator.version>2.2.14</json.schema.validator.version>
-        <groovy.version>2.5.11</groovy.version>
-        <junit.version>4.13.1</junit.version>
-        <lombok.version>1.18.18</lombok.version>
-        <mockito.version>3.9.0</mockito.version>
+        <groovy.version>3.0.8</groovy.version>
+        <junit.version>4.13.2</junit.version>
+        <lombok.version>1.18.20</lombok.version>
+        <mockito.version>3.10.0</mockito.version>
         <rx.version>1.3.8</rx.version>
-        <slf4j-api.version>1.7.30</slf4j-api.version>
+        <slf4j-api.version>2.0.0-alpha1</slf4j-api.version>
+        <spring-test.version>5.3.7</spring-test.version>
 
-        <maven.plugin-tools.version>3.3</maven.plugin-tools.version>
-        <maven.version>3.3.3</maven.version>
+        <maven.plugin-tools.version>3.6.1</maven.plugin-tools.version>
+        <maven.version>3.8.1</maven.version>
         <maven.source-plugin.version>2.2.1</maven.source-plugin.version>
         <maven.jar-plugin.version>3.0.2</maven.jar-plugin.version>
         <maven.javadoc-plugin.version>2.9.1</maven.javadoc-plugin.version>
         <maven.checkstyle-plugin.version>3.1.0</maven.checkstyle-plugin.version>
         <maven.aspectj-plugin.version>1.12.6</maven.aspectj-plugin.version>
 
-        <google.jsr305.version>3.0.0</google.jsr305.version>
-        <free.port.finder.version>1.0</free.port.finder.version>
+        <google.jsr305.version>3.0.2</google.jsr305.version>
+        <free.port.finder.version>1.1.1</free.port.finder.version>
         <jjwt.version>0.9.1</jjwt.version>
         <reflections.version>0.9.12</reflections.version>
-        <azure.core.version>1.15.0</azure.core.version>
-        <azure.core-http-netty.version>1.9.1</azure.core-http-netty.version>
-        <nimbusds.oauth2.oidc.sdk.version>7.1.1</nimbusds.oauth2.oidc.sdk.version>
-        <okhttp.version>4.9.1</okhttp.version>
+        <azure.core.version>1.16.0</azure.core.version>
+        <azure.core-http-netty.version>1.9.2</azure.core-http-netty.version>
+        <nimbusds.oauth2.oidc.sdk.version>9.7</nimbusds.oauth2.oidc.sdk.version>
+        <okhttp.version>5.0.0-alpha.2</okhttp.version>
```

Comment:
Do we need to upgrade the major version with alpha?

Reply:
reverted.